playbooks/cluster/openshift/vars/gluster.yml

```diff
@@ -0,0 +1,11 @@
+# Namespace to create for the GlusterFS and heketi resources
+openshift_storage_glusterfs_namespace: app-storage
+# Do not create a StorageClass; a custom StorageClass will be created via APB
+# for use by kubevirt
```

Comment:
Hmm,... to complicated. I don't follow it.
`Do not create a StorageClass ; a custom StorageClass will be created via APB for use by kubevirt ` is true when `openshift_storage_glusterfs_storageclass: false`. And what happens when this var is true? 

Is a comment `Use APB to create a custom StorageClass for KubeVirt.` equivalent?

Reply:
When false, a StorageClass will not be created. When true, a StorageClass will be created. The rest of the comment is explaining why we're setting that to false, namely that a different, custom StorageClass will be related later. Your suggestion implies that setting this to false creates another custom StorageClass, which is not accurate. I'm not sure how else to best combine those two statements.